app/WebParsing/ArtSciParser.hs

```diff
@@ -46,7 +46,10 @@ getCalendar str = do
         coursesSoup = lastH2 tags
         course = map (processCourseToData . filter isTagText) $ partitions isCourseTitle coursesSoup
     print $ "parsing " ++ str
+    --runSqlite databasePath $
```

Comment:
Right, I looked into this even more, but still was not able to come up with a solution. I don't think we should worry about this for now, so you can revert the changes in this file.